index.html

```diff
@@ -9205,6 +9284,24 @@ <h2>Index</h2>
 -->
 
 <dl>
+ <dt>ARIA
+ <dd>
+   <p>The following terms are defined
+     in the Accessible Rich Internet Applications (WAI-ARIA) 1.2 specification: [wai-aria-1.2]
+     <ul>
+       <li><dfn><a href="https://w3c.github.io/aria/#introroles">WAI-ARIA role</a></dfn>
+     </ul>
+
+  <dt>Accessible Name and Description Computation
```

Comment:
Generally we tend to list related specs under a single topic, so this be a `<dd>` under `<dt>Aria` from above.